en/docs/fragments/add-custom-attributes-saml.md

```diff
@@ -0,0 +1,15 @@
+1.  Navigate to the **Attributes** tab of the application you registered in the previous step. 
+
+2.  Enable attribute mapping by clicking on the toggle button, **Enable mapping**. 
+
+3.  Click on the edit icon. 
+
+4.  Choose the attributes that you require for this application. For this scenario, we have chosen `Country`, `Username`, and `Email`. 
```

Comment:
Choose --> Select 

(generally we've used the word "select" when it comes to dropdown lists)

Reply:
For this scenario, select `Country`,....